modules/sdk/gradle-plugins-workspace/src/main/resources/com/liferay/gradle/plugins/workspace/task/dependencies/templates/language/client-extension.yaml

```diff
@@ -0,0 +1,16 @@
+assemble:
+    - fromTask: generateLanguageBatchEngineData
+      into: batch
+liferay-language-batch:
```

Comment:
More of a product question, but do we actually want to prefix this (and the one below) with `liferay-`? This is not a sample, it will be used by customers.

Reply:
I took the liberty to remove the prefix. I believe it unnecessary. @ethib137, if you think it's necessary, please let me know and I'll remove the commit